assets/src/design-system/theme/helpers/outline.js

```diff
@@ -23,11 +23,14 @@ import { css } from 'styled-components';
  */
 import { FOCUS_VISIBLE_SELECTOR } from '../global';
 
-export const focusableOutlineCSS = (accent) => css`
-  border: solid 2px transparent;
+export const focusableOutlineCSS = (v) => {
```

Comment:
i like this!

Reply:
I just use `v` to indicate a generic value. I can update it to `colorOrProps` tho to make it more descriptive